### ISSUE TITLE
E-Mail: Reminder

### ISSUE THREAD

**Issue by Tosh79** [Tosh79/lamonee_backlog#116]
*lamonee soll monatlich eine Reminder-E-Mail an die Nutzer schicken, um sie daran zu erinnern, dass ihr Budget und ihre Konto-Transaktionen gepflegt werden müssen.

**Comment by Tactticus** [comment]
Ja, bitte. Das wäre wirklich ein gutes kleines Feature. Würde ich begrüßen und nutzen.
